Web/HTML/core-html-elements/update-hyperlinks.md

```diff
@@ -52,36 +128,21 @@ How is a placeholder link created, where the link works, but it doesn't open a n
 
```

Comment:
You can't have two answer sections, all the answers have to be at the bottom. Since there are two slots for two questions, just make sure the first two answers are the correct ones